packages/react/src/components/ComboBox/ComboBox-test.js

```diff
@@ -1431,5 +1431,52 @@ describe('ComboBox', () => {
       expect(screen.getByTestId('change-count').textContent).toBe('1');
       expect(findInputNode()).toHaveDisplayValue('Option 2');
     });
+
+
```

Comment:
This file isn't formatted. Did you see any errors creating the commit? The pre-commit hook should have formatted this file unless commit verfication was bypassed.